sys/net/gnrc/netif/lorawan/gnrc_netif_lorawan.c

```diff
@@ -255,7 +256,7 @@ static int _send(gnrc_netif_t *netif, gnrc_pktsnip_t *payload)
     { .type = netif->lorawan.ack_req ? MCPS_CONFIRMED : MCPS_UNCONFIRMED,
       .data =
       { .pkt = (iolist_t *)payload, .port = netif->lorawan.port,
-          .dr = netif->lorawan.datarate } };
+          .dr = netif->lorawan.datarate, .adr = netif->lorawan.adr } };
```

Comment:
Same here.This should be setup using NETOPTs.

Reply:
Done. Removed